modules/elements.js

```diff
@@ -346,7 +346,14 @@ class ElementsModule {
 		fluxloaderAPI.setPatch("js/336.bundle.js", "corelib:conveyorBeltIgnores", {
 			type: "replace",
 			from: `d=[a.RJ.Water,a.RJ.Steam,a.RJ.Lava`,
-			to: `d=[${prependJoin("a.RJ.", this.#otherFeatures.conveyorBeltIgnores)}`,
+			to: `d=[${prependJoin("a.RJ.", this.otherFeatures.conveyorBeltIgnores)}`,
+		});
+		//all the following patches improve the movement of particles too colide with all recipe blocks
+		//used chatgpt for this cuz i don't wanna learn regex, STUPID CHATBOTS CHANGING CODE
+		fluxloaderAPI.setPatch("js/515.bundle.js", "corelib:solidMovementTweaks", {
+			type: "regex",
+			pattern: "R=i\\.Both,\\s*F=\\(0,o\\.Ol\\)\\(g\\);\\s*if\\(p\\|\\|M\\|\\|\\(0,o\\.W\\)\\(g,a\\.vZ\\.Block\\)\\)([\\s\\S]*?)if\\(\\(!g\\|\\|M\\|\\|!S\\(e,t,g\\)\\)\\s*&&\\s*!\\(0,f\\.v\\)\\(e,t,g\\)\\)",
```

Comment:
As horrid-looking as the regex is, it's functional and idk how to improve it. I do have a slight cleanup, since we don't need the `\s*` since we know there's no whitespace.
```suggestion
			pattern: "R=i\\.Both,F=\\(0,o\\.Ol\\)\\(g\\);if\\(p\\|\\|M\\|\\|\\(0,o\\.W\\)\\(g,a\\.vZ\\.Block\\)\\)([\\s\\S]*?)if\\(\\(!g\\|\\|M\\|\\|!S\\(e,t,g\\)\\)&&!\\(0,f\\.v\\)\\(e,t,g\\)\\)",
```